tests/fixtures/math_guy/forwarded_sly_data.hocon

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+# This file defines everything necessary for a data-driven test.
+{
+    # Describes what agent to test against.
+    "agent": "math_guy_passthrough",
```

Comment:
Test case hocon that looks for all the allow.to/from_down/upstream cases for sly_data.